middleware/passport-config.ts

```diff
@@ -5,214 +5,50 @@
 
 'use strict';
 
-import { IProviders } from "../transitional";
-import { Operations } from "../business/operations";
+import createAADStrategy from "./passport/aadStrategy";
```

Comment:
Only to have a discussion, not an issue: I really struggle with casing variables including `aad` and `github`. It is not always inconsistent.

Questions are, is it really...

- `githubVariable`
- `gitHubVariable`
- `aadVariable`
- `AADVariable`

I think the consensus I've been pointed at by others is that GitHub as a variable name is essentially just `github` with proper casing, similar to the domain name; so, `githubLogin`, `githubStrategy`, or, in this case, `createGithubStrategy`, though I really do not like that look, so maybe it's OK to sacrifice that appearance to align with `GitHub` capitalization?

Am I being super pedantic on this?

Reply:
Not overly pedantic; or at least, my type of pedantic! 😄 

I think you are correct on `github`/`Github`. The one I'm a bit more torn on is `aad`. Given that the convention here is `lowerCamelCase`, I think it's appropriate to have `aadVariable`, but `createAADStrategy` is the potentially wonky one. Straight out of [Microsoft's own recommendations for .NET](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN), this should likely be `createAadStrategy`. But to my eye, that just looks...not quite right. I'm happy to go with your call on this one!

Reply:
Let's keep it as you have it for now. I'm with you... silly edge case I suppose, but capitalized AAD looks better here.